system_tests/storage_trie_test.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2025, Offchain Labs, Inc.
+// For license information, see:
+// https://github.com/OffchainLabs/nitro/blob/master/LICENSE.md
+
+package arbtest
+
+import (
+	"context"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/ethereum/go-ethereum/core/rawdb"
+
+	"github.com/offchainlabs/nitro/util/arbmath"
+	"github.com/offchainlabs/nitro/validator/valnode"
+)
+
+var withL1 = true
```

Comment:
This withL1 variable is global to the system_tests package, so I think we should just inline it

Reply:
Done.